### PR TITLE
Table UI test cases

### PR DESCRIPTION
**Pull request by arslanhaiderbuttar**
## Title
Table Widget UI testcases.

## Description
Widget : Table 

Fixes # (issue)

- On Search Text Change Action 
- Default array data 
- On Row Selected Action 
- Explore Widget related documents  
- On Page Change Action
- Default search text 
- Default selected row 
- Delete the Table
- Change Column data type to Image
- Change Column data type to URL
- Row Height Functionality
- Drage and Drop Table widget
- Verify Default data
- Filter The Data using 'Empty'  Filter The Data using 'Not Empty'

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

## Test coverage results :test_tube:
<details><summary>:green_circle: Total coverage has increased</summary>

    // Code coverage diff between base branch:release and head branch: TableUITestCases 
Status | File | % Stmts | % Branch | % Funcs | % Lines 
 -----|-----|---------|----------|---------|------ 
 :green_circle: | total | 54.62 **(0.01)** | 36.71 **(0.01)** | 33.06 **(0)** | 55.23 **(0.01)**
 :green_circle: | app/client/src/utils/autocomplete/TernServer.ts | 50.71 **(0.24)** | 38.6 **(0.88)** | 36.21 **(0)** | 55.35 **(0.27)**</details>

### CODE REVIEW

File: app/client/cypress/integration/Smoke_TestSuite/ClientSideTests/DisplayWidgets/Table_PropertyPane_spec.js [hunk 1]
```diff
@@ -10,91 +10,226 @@ describe("Table Widget property pane feature validation", function() {
...
+    cy.get(publish.backToEditor).click();
+  });
+
+  it("Verify default array data", function() {
+    // Open property pane
+    cy.SearchEntityandOpen("Table1");
+    // Open Widget side bar
+    cy.get(widgetsPage.addWidget).click();
+    // Drag and drop table widget
+    cy.dragAndDropToCanvas("tablewidget", { x: 300, y: 200 });
```
Comment (NandanAnantharamu) [inline app/client/cypress/integration/Smoke_TestSuite/ClientSideTests/DisplayWidgets/Table_PropertyPane_spec.js:36]: This drag and drop will become flaky, can you check this before we push by running multiple times.

File: app/client/cypress/support/commands.js [hunk 2]
```diff
@@ -2074,6 +2104,16 @@ Cypress.Commands.add("runAndDeleteQuery", () => {
...
 Cypress.Commands.add("dragAndDropToCanvas", (widgetType, { x, y }) => {
   const selector = `.t--widget-card-draggable-${widgetType}`;
+  cy.get(selector)
+    .trigger("mousedown", { button: 0 }, { force: true })
+    .trigger("mousemove", x, y, { force: true });
+  cy.get(explorer.dropHere)
+    .trigger("mousemove", x, y)
+    .trigger("mouseup", x, y);
+});
+
```
Comment (NandanAnantharamu) [inline app/client/cypress/support/commands.js:2114]: If this working without any issue, we can go-ahead

**Review by NandanAnantharamu — DISMISSED** [review]
Approved for test run

**Review by yatinappsmith — APPROVED** [review]

**Review by NandanAnantharamu — APPROVED** [review]
LGTM

File: app/client/cypress.json [hunk 3]
```diff
@@ -4,7 +4,6 @@
   "requestTimeout": 21000,
   "pageLoadTimeout": 20000,
   "video": true,
-  "videoUploadOnPasses": false,
```
Comment (yatinappsmith) [inline app/client/cypress.json:7]: Why have you made changes to cypress.json?

**Review by NandanAnantharamu — DISMISSED** [review]
cypress.json file has been changed, can you remove it ?

**Comment by arslanhaiderbuttar** [comment]
> cypress.json file has been changed, can you remove it ?

It's done. can you approve again. @NandanAnantharamu

**Review by yatinappsmith — APPROVED** [review]

**Review by yatinappsmith — APPROVED** [review]
LGTM

**Pull request merged**